Source/Krypton Components/Krypton.Ribbon/Group Contents/KryptonRibbonGroupComboBox.cs

```diff
@@ -413,8 +413,7 @@ public Size MaximumSize
         [Category(@"Appearance")]
         [Description(@"Text associated with the control.")]
         [Editor(typeof(MultilineStringEditor), typeof(UITypeEditor))]
-        [AllowNull]
-        public string? Text
+        public string Text
```

Comment:
If you look in the Decompiled source for net 6 onwards, I think this can return null.

Reply:
Underlying base properties are all non-nullable string. Older code of course.

Winforms.Control.Text 
	non-nullable
	gets: always a string
	sets: an empty string if value is null
	
Winforms.ComboBox: (inherits from Control.Text)
	non-nullable
	gets: always a string
	sets: can take a null value (which make no sense in relation to Control.Text's behaviour)

Either leave it as is now, since under water it will always set to a string of return at least an empty string.
So null make no sense. And other solutions are imo merely a work-around.

Or allow null input on a inherited non-nullable type like this.
```cs
        /// <summary>
        /// Gets and sets the text associated with the control.
        /// </summary>
        [Category(@"Appearance")]
        [Description(@"Text associated with the control.")]
        [Editor(typeof(MultilineStringEditor), typeof(UITypeEditor))]
        [AllowNull]
        public string Text
        {
            get => ComboBox.Text;
            set => ComboBox.Text = value!;
        }
```